.eslintrc

```diff
@@ -14,7 +14,8 @@
     "consistent-return": 0,
     "comma-dangle": 0,
     "spaced-comment": 0,
-    "global-require": 0
+    "global-require": 0,
+    "no-param-reassign": ["error", {"props": false}]
```

Comment:
Also, just to clarify, this change still doesn't allow reassigning params, it only allows mutating properties of params.